packages/sui-deploy/README.md

```diff
@@ -60,15 +101,16 @@ $ sui-deploy spa --help
   "version": "1.0.0",
   "scripts": {
     "build": "sui-bundler build -C",
-    "deploy": "sui-deploy test-project --now"
+    "deploy": "sui-deploy test-project --now -a 'my-user:my-password'"
```

Comment:
Which is the default public folder?!

Reply:
https://github.com/SUI-Components/sui/pull/276/files#diff-304d634116a986f8a14b512c69a22832R23